src/TimeService.Validation/LeaveTime/CreateLeaveTimeRequestValidator.cs

```diff
@@ -1,19 +1,57 @@
 ﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Common;
 using LT.DigitalOffice.TimeService.Data.Interfaces;
 using LT.DigitalOffice.TimeService.Models.Dto.Filters;
 using LT.DigitalOffice.TimeService.Models.Dto.Requests;
 using LT.DigitalOffice.TimeService.Validation.LeaveTime.Interfaces;
+using MassTransit;
+using Microsoft.Extensions.Logging;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace LT.DigitalOffice.TimeService.Validation.LeaveTime
 {
     public class CreateLeaveTimeRequestValidator : AbstractValidator<CreateLeaveTimeRequest>, ICreateLeaveTimeRequestValidator
     {
-        public CreateLeaveTimeRequestValidator(ILeaveTimeRepository repository)
+        private readonly IRequestClient<ICheckUsersExistence> _rcCheckUsersExistence;
+        private readonly ILogger<CreateLeaveTimeRequestValidator> _logger;
+
+        private bool CheckUserExistence(List<Guid> userIds)
         {
+            string logMessage = "Cannot check existing users {userIds}";
+
+            try
+            {
+                IOperationResult<ICheckUsersExistence> response = _rcCheckUsersExistence.GetResponse<IOperationResult<ICheckUsersExistence>>(
+                    ICheckUsersExistence.CreateObj(userIds)).Result.Message;
+                if (response.IsSuccess && response.Body.UserIds.Count == 1)
```

Comment:
maybe you should add check that id in request and in response are equal

Reply:
done